tests/integration/test_examples.py

```diff
@@ -0,0 +1,83 @@
+"""
+End to end test of String load from examples/string
+
+"""
+
+import os
+
+import pytest
+from kgx.cli.cli_utils import validate
+
+from koza.koza_runner import transform_source
+from koza.model.config.source_config import OutputFormat
+
+
+@pytest.mark.parametrize(
+    "ingest, output_names, output_format",
+    [
+        (
+            "string",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.tsv,
+        ),
+        (
+            "string",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.jsonl,
+        ),
+        (
+            "string-declarative",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.tsv,
+        ),
+        (
+            "string-declarative",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.jsonl,
+        ),
+        (
+            "string-w-map",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.tsv,
+        ),
+        (
+            "string-w-map",
+            [
+                "stringdb.protein-links-detailed",
+            ],
+            OutputFormat.jsonl,
+        ),
+    ],
+)
+def test_examples(ingest, output_names, output_format):
+
+    source = f"examples/{ingest}/metadata.yaml"
+    output_suffix = str(output_format).split('.')[1]
+    output_dir = f"./test-output/{ingest}-{output_suffix}"
+
+    output_files = [
```

Comment:
consider simplifying to:

    output_files = [
        output_file
        for file in output_names
        for output_file in [
            f"{output_dir}/{file}_nodes.{output_suffix}",
            f"{output_dir}/{file}_edges.{output_suffix}",
        ]
    ]
```

but either works!